QUARKUS-874.md

```diff
@@ -2,17 +2,24 @@
 
 JIRA link: https://issues.redhat.com/browse/QUARKUS-874
 
-The goal is to ensure some level of testing and identify existing limitations on Quarkus side for Windows.
+The goal is to provide production support for JVM mode on Windows, both on bare-metal instances and in OpenShift.
```

Comment:
I think for now the only support for OpenShift we want is verifying that deployment from Windows works, correct? We don't aim specifically to have OCP worker nodes running on Windows machines.

Reply:
Alright, I didn't understand what is requirement for the OpenShift. Updated text, thank you.